src/DrCabal/Profile.hs

```diff
@@ -126,6 +129,13 @@ entriesToPhase start times = Phase
             Nothing -> it `minusw` start
             Just bt -> it `minusw` bt
 
+    calcHaddock :: Word64
+    calcHaddock = case building of
```

Comment:
The logic for `calcBuilding` needs to be updated as well. It needs to check if we have the `Haddock` phase and calculate the difference between the `Haddock` and `Building` in this case. Otherwise, the Build time will include the time needed to build the `Haddock`

Reply:
My bad. Changed the order of tags. :joy: